index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="style.css" />
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" as="style" crossorigin="" href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css">
+    <title>판다마켓</title>
+</head>
+
+<body>
+    <div class="header">
+        <div class="header_contents">
+            <a href="/"><img src="./Group 19.png"></a>
+            <div class="header_login">
+                <a class="link_color" href=".login">로그인
+            </div>
+        </div>
+    </div>
+    <div class="banner">
+        <div class="banner_contents">
+            <div class="banner_looking">
+                <div class="banner_text">일상의 모든 물건을<br>
+                    거래해 보세요</div>
+                <div class="looking"><a class="link_color" href="/items">구경하러 가기</a></div>
+            </div>
+            <img src="./Img_home_top.png">
+        </div>
+    </div>
+    <div>
+        <div class="main">
+            <div class="main_in">
+                <img src="./Frame 2608833.png">
+                <div class="items_first">
+                    <div class="items_first1">Hot Item</div>
+                    <div class="items_first_text">
+                        <div class="items_first2">인기 상품을<br>
+                        확인해보세요</div>
+                        <div class="items_first3">가장 HOT한 중고거래 물품을<br>
+                        판다 마켓에서 확인해 보세요</div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="main">
+            <div class="main_in">
+                <div class="items_second">
+                    <div class="items_second1">Search</div>
+                    <div class="items_second_text">
+                        <div class="items_second2">구매를 원하는<br>
+                            상품을 검색하세요</div>
+                        <div class="items_second3">구매하고 싶은 물품은 검색해서<br>
+                            쉽게 찾아보세요</div>
+                    </div>
+                </div>
+                <img src="./Img_home_02.png">
+            </div>
+        </div>
+        <div class="main">
+            <div class="main_in">
+                <img src="./Img_home_03.png">
+                <div class="items_third">
+                    <div class="items_third1">Register</div>
+                    <div class="items_third_text">
+                        <div class="items_third2"> 판매를 원하는<br>
```

Comment:
❗️ 수정요청
공백은 의도치 않은 동작을 발생시킬 수 있습니다~ 
필요한 공백이 아니면 가능하면 습관적처럼 없애는 것을 추천드려요~

```suggestion
                        <div class="items_third2">판매를 원하는<br>
```